core/domain/feedback_services.py

```diff
@@ -172,3 +188,99 @@ def get_thread_analytics(exploration_id):
     """
     return feedback_jobs.FeedbackAnalyticsAggregator.get_thread_analytics(
         exploration_id)
+
+
+def create_suggestion(exploration_id, author_id, exploration_version,
+                      state_name, suggestion_content):
+    """Creates a new SuggestionModel object and the corresponding
+    FeedbackThreadModel object."""
+
+    thread_id = _create_models_for_thread_and_first_message(
+        exploration_id, state_name, author_id, 
+        DEFAULT_SUGGESTION_THREAD_SUBJECT, EMPTY_TEXT, True)
```

Comment:
Note for the future: if you're passing args which already have default values, include the name of the arg. E.g.

    ..., EMPTY_TEXT, has_suggestion=True)

(Though this may be obsolete here because I suggested above that has_suggestion doesn't really need to be a default arg.)